packages/java/maven-plugin/src/main/java/com/vaadin/fusion/maven/ParserConfiguration.java

```diff
@@ -22,12 +30,17 @@
         return Optional.ofNullable(openAPIPath);
     }
 
-    public Optional<PluginList> getPlugins() {
+    public Optional<Plugins> getPlugins() {
         return Optional.ofNullable(plugins);
     }
 
     public static class Plugin {
```

Comment:
I'm trying to maintain an approach where I keep only the top-level structure package-private since all nested classes/methods will become package-private even if they are declared as `public`.